packages/vm/package.json

```diff
@@ -42,7 +45,7 @@
     "prepublishOnly": "../../config/cli/prepublish.sh && npm run test:buildIntegrity",
     "profiling": "0x ./benchmarks/run.js profiling",
     "test": "echo \"[INFO] Generic test cmd not used. See package.json for more specific test run cmds.\"",
-    "test:API": "npx vitest run ./test/api/",
+    "test:API": "npx vitest run -c ./vitest.config.ts ./test/api/",
```

Comment:
Ah, had to check but this reads from the package config which expands the base config in the root of the monorepo. Which also makes me realize that I think we have more of these package configs.

Reply:
I had expected one in EVM but it's not there, so all looks good!